scripts/faces/triangulation_planeface.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""
+
+"""
+
+import volmdlr as vm
+import volmdlr.faces as vmf
+import volmdlr.stl as vmstl
+
+list_tri = []
+for stl_file in [
+                'face0.stl',
+                'face1.stl'
+                  ]:
+    file_faces = vmstl.Stl.from_file(stl_file)
+    triangles = file_faces.triangles
+    list_tri.append(triangles)
+    # for face in file_faces.triangles:
+    #     points = face.triangulation().points
+    #     for three_pos in face.triangulation().triangles :
+    #         triangles.append(vmf.Triangle3D(points[three_pos[0]], 
+    #                                         points[three_pos[1]],
+    #                                         points[three_pos[2]]))
+  
+for triangles in list_tri:
+    print()          
+    for tri in triangles : 
```

Comment:
What is the point of this script? Show ZeroDivision Error?

Reply:
yes, that is correct. Its is a test case to show this error

Reply:
as it is not a bug but a feature we can remoce this script and the face0.stl and face1.stl files

Reply:
alright